webapp/src/css/inbox.less

```diff
@@ -1299,6 +1391,28 @@ mm-sidebar-menu .mat-sidenav-container {
   }
 
   .app-root:not(.old-nav) {
+    &.messages,
+    &.tasks,
+    &.reports,
+    &.contacts {
+      .loading-status {
+        margin: 10px 10px calc(@tab-navbar-size + 10px);
```

Comment:
The privacy policy on mobile isn't showing the full page, it needs more margin bottom
![Screenshot 2024-09-24 at 11 15 25 AM](https://github.com/user-attachments/assets/53cbac12-8707-469e-ba51-32595b518d47)